remote-helloworld-mdb/configure-remote-broker.cli

```diff
@@ -10,4 +10,5 @@ echo Adding configuration to connect to remote broker
 # Ensure that the resource adapter to the remote broker is used by default by EJBs
 /subsystem=ejb3:write-attribute(name=default-resource-adapter-name, value=RemoteConnectionFactory)
 # Ensure that the default resource adapter used for JMS is the resource adapter to the remote broker
-/subsystem=ee/service=default-bindings:write-attribute(name=jms-connection-factory, value=java:jboss/RemoteConnectionFactory)
\ No newline at end of file
+/subsystem=ee/service=default-bindings:write-attribute(name=jms-connection-factory, value=java:jboss/RemoteConnectionFactory)
+reload
```

Comment:
I think you need to remove this reload, cause such command doesn't work well with any provisioning

Reply:
Nevermind, false alarm, this script is not executed when provisioning.

Reply:
The reload is not required. In The README is manual step for stop and review the configuration. Then it is followed by a start command.